### ISSUE TITLE
Black screen using exoplayer

### ISSUE THREAD

**Issue by snti** [Samsung/GearVRf#1218]
Hi guys,

I have an app, this app starts with a video, at the end of the video there is a gallery with images.
I'm using the exoplayer to reproduce the video. 
Sometimes when I start the app the sound is working, but I can´t see anything, the display is totally black. When I close the app and open it again the app works!

Any idea?

Regards!

**Comment by thomasflynn** [comment]
sounds like a pause and resume issue with the exoplayer?  any thoughts @liaxim

**Comment by liaxim** [comment]
Yes, or some kind of race condition in the app's initialization. You can take a look at our gvr-video sample which by default uses ExoPlayer - in case we might be doing something different. You could share with us a sample that reproduces the problem - then we might be able to help more.

**Comment by liaxim** [comment]
@snti Do you have anything for us to look at?
